### ISSUE TITLE
Adding in a search feature to the API

### ISSUE THREAD

**Issue by RogueSergeant** [LibraryOfCongress/api.congress.gov#105]
As a Congress API user, I want to be able to get a limited subset of packages or granules based on a query.

Requested functions:

    search by keyword or specific search operators (similar to existing congress.gov ui)
    allow user to specify fields returned (based on availability within index)
    field names should follow summary json naming convention
    specify whether response should be at package, granule level, or both

**Comment by mnewatloc** [comment]
We've added this to our list of possible future enhancements.

**Comment by akhilrajEth** [comment]
Is there any update to the status of this enhacement?

**Comment by apreiter18** [comment]
@akhilrajEth - Good morning. At this time, we do not have an update on this. Thanks.

**Comment by pding2002** [comment]
We are looking for the search feature on Title. Any update?

**Comment by apreiter18** [comment]
No, we do not have an update. If we are able to do this, any updates would be posted in the change log.

**Comment by apreiter18** [comment]
Issue #314 is where updates to this issue will be posted. We are going to investigate the feasibility of this. Updates will be posted in that issue and in the API's change log.
